client/src/partials/fiscal/fiscal.new.js

```diff
@@ -0,0 +1,70 @@
+angular.module('bhima.controllers')
+.controller('FiscalCreateController', FiscalCreateController);
+FiscalCreateController.$inject = [
+    '$state', 'ScrollService', 'FiscalService', 'NotifyService', 'util'
+];
+
+/**
+ * This controller is responsible for creating a debtor group. It provides
+ * utility functions for submission and error handling.
+ *
+ * @todo refactor code to remove redundant features introduced previously
+ *
+ * @module fiscal/new
+ */
+function FiscalCreateController($state, ScrollTo, fiscalService, Notify, util) {
+  var vm = this;
+  // expose state for optional view elements
+  vm.state   = $state;
+  vm.fiscal  = {};
+  vm.endDate = endDate;
+
+  vm.maxLength = util.maxTextLength; 
+
+  function endDate(){
+    var start_date = new Date(vm.fiscal.start_date);
+    var previousDay;
+    vm.end_date = start_date.setMonth(start_date.getMonth() + vm.fiscal.number_of_months);
```

Comment:
Does this work properly with fiscal years that are more than 12 months?

According to #440, we should use `moment()` for adding, removing, and calculating dates.  [Here](http://momentjs.com/docs/) is the documentation.